src/listeners/instances/ham/bq.ts

```diff
@@ -7,6 +7,11 @@ export enum DeltaOperation {
 	Deleted = "deleted",
 }
 
+interface FidWithAddress {
```

Comment:
nit: `FID` everywhere we can, so 
```suggestion
interface FIDWithAddress {
```